### PR TITLE
LPS-147643

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-147643

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Review by naraandrade — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/actions/ActionsInfo.js [hunk 1]
```diff
@@ -41,34 +41,36 @@ const ActionsInfo = ({identifier, index, sectionsLength, setSections}) => {
...
+		!executionTypeOptions
+			.map((option) => option.value)
+			.includes('onAssignment')
+	) {
+		executionTypeOptions.push({
+			label: Liferay.Language.get('on-assignment'),
+			value: 'onAssignment',
+		});
+	}
+	sortElements(executionTypeOptions, 'value');
```
[the remark this answers is not in the document]
Comment (alinedoleron) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/actions/ActionsInfo.js:60]: Oh no! Sorry for this mistake!! Thanks!!

**Comment by alinedoleron** [comment]
ci:test:sf

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**